index.js

```diff
@@ -99,10 +102,27 @@ export class Placement {
       this.target.appendChild(element);
 
       return this;
+    }).then((placement) => {
+      // Detect when the ad is in the viewport
+      // Add the view pixel to the DOM to count the view
+
+      var viewport_detection = setInterval((element) => {
```

Comment:
While `var` works here, `let` reduces the scope of the variable, which would be function level with `var` -- `let` is block level scoped.

```suggestion
      let viewport_detection = setInterval((element) => {
```